code/game/gamemodes/blob/overmind.dm

```diff
@@ -48,13 +48,14 @@
 	update_health_hud()
 
 /mob/camera/blob/update_health_hud()
-	if(blob_core)
+	if(blob_core && hud_used)
 		hud_used.blobhealthdisplay.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#e36600'>[round(blob_core.obj_integrity)]</font></div>"
 
 /mob/camera/blob/proc/add_points(var/points)
 	if(points != 0)
 		blob_points = Clamp(blob_points + points, 0, max_blob_points)
-		hud_used.blobpwrdisplay.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#82ed00'>[round(src.blob_points)]</font></div>"
+		if(hud_used)
```

Comment:
Do you maybe want to to fold it with `if` above? (i don't know)

Reply:
Fold it? As in add it to the points if? If so then no seeing the blobs points do need to be added on line 56

Reply:
You're right, ignore my comment.